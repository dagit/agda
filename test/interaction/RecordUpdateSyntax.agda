module RecordUpdateSyntax where

data ⊤ : Set where
  tt : ⊤

record R : Set where
  field
    a b : ⊤

test : R
test = record {!!} { a = tt }
