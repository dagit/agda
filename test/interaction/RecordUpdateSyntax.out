Prelude> Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking RecordUpdateSyntax (RecordUpdateSyntax.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished RecordUpdateSyntax.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : R\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-give-action 0 "record { a = tt; b = tt }")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 